src/modules/search/ObjectDetailPage.tsx

```diff
@@ -1546,7 +1540,7 @@ export const ObjectDetailPage: FC<DefaultSeoInfo> = ({ title, description, image
 
 	const renderPageContent = () => {
 		if (mediaInfoIsLoading || visitRequestIsLoading || visitorSpaceIsLoading) {
-			return <Loading fullscreen owner="object detail page: render page content" />;
+			return <Loading fullscreen owner={'object detail page: render page content'} />;
```

Comment:
Does SSR require dynamic props even for static values?

Reply:
no this is a remmanent for some testing code i had
fixed